src/main/scala/wdl4s/expression/WdlStandardLibraryFunctions.scala

```diff
@@ -1,7 +1,7 @@
 package wdl4s.expression
 
 import wdl4s.types._
-import wdl4s.values._
+import wdl4s.values.{WdlArray, WdlInteger, WdlValue, _}
```

Comment:
This is IntelliJ mischief, not a conscious choice on my part.  I'll put it back if IntelliJ allows me.